x-pack/test/fleet_api_integration/apis/agent_policy/agent_policy.ts

```diff
@@ -513,8 +513,16 @@ export default function (providerContext: FtrProviderContext) {
           })
           .expect(200);
 
-        // add endpoint package policy, which is required for tamper protection
         await supertest
+          .post(`/api/fleet/epm/packages/endpoint/8.10.2`)
+          .set('kbn-xsrf', 'xxxx')
+          .send({
+            force: true,
+          })
+          .expect(200);
+
+        // add endpoint package policy, which is required for tamper protection
+        const res = await supertest
```

Comment:
@juliaElastic  - this line is causing a legit typescript error, because of the unused variable. 

However, if the tests are passing outside this typecheck, we can apply this suggestion, and I'll admin-merge the PR without another whole CI run (to speed up the fix).

So, don't accept this before we've seen the initial CI run finish: https://buildkite.com/elastic/kibana-pull-request/builds/164366#018aefc7-0ef4-43eb-b4b5-3aec6c95b43d

```suggestion
        await supertest
```

Reply:
thanks for raising, I left this in unintentionally (logged out the error locally before)